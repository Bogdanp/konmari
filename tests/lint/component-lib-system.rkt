#lang racket/base

(require component)

(provide dev-system)

(define-system dev)

(define-system prod)
