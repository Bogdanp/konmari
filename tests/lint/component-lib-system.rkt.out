/component-lib-system.rkt:9:16:warning:identifier 'prod-system' is never used
